src/actions/smartWalletActions.js

```diff
@@ -823,6 +825,7 @@ export const onSmartWalletSdkEventAction = (event: Object) => {
         && [ARCHANOVA_PPN_PAYMENT_COMPLETED, ARCHANOVA_PPN_PAYMENT_PROCESSED].includes(txStatus)) {
         const paymentInfo = `${formatMoney(txAmountFormatted.toString(), 4)} ${txToken}`;
         if (txStatus === ARCHANOVA_PPN_PAYMENT_COMPLETED) {
+          isLogV2AppEvents() && dispatch(logEventAction('v2_transaction_received'));
```

Comment:
Wrong place, this is a transaction for the Archanova service which is now obsolete. You will need to add this where we receive transaction events from the Etherspot SDK.